lib/board.rb

```diff
@@ -124,25 +80,23 @@ def render_horizontal
     string_part_1 = "  "
     string_part_2 = []
     string_part_3 = "\n"
-    horiz_coord.each do |num|
-    string_part_2 << "#{num} "
-    end
-    return string_part_1 + string_part_2.join.to_s + string_part_3
+    horiz_coord.each { |num| string_part_2 << "#{num} " }
+    (string_part_1 + string_part_2.join.to_s + string_part_3)
```

Comment:
This one required the parentheses because the last line is being evaluated, so it needs to know to return the entire string. :)